api/axios.ts

```
@@ -4,6 +4,7 @@ import {
   setClerkToken,
 } from "@/store/tokenStore";
 import { forceLogout } from "@/utils/forceLogout";
+import { forceLogout } from "@/utils/forceLogout";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'forceLogout'. Consider to delete it or rename it.

'forceLogout' is defined here:


(lint/suspicious/noRedeclare)